api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -2595,6 +2597,12 @@ type HostedClusterStatus struct {
 	// +patchStrategy=merge
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
 
+	// Payload represents the CPU architecture type of the HostedCluster.Spec.Release.Image
+	// +kubebuilder:validation:Enum=multi;arm64;amd64;ppc64le
```

Comment:
Enum values in Kube APIs should be pascal case
```suggestion
	// +kubebuilder:validation:Enum=Multi;ARM64;AMD64;PPC64LE
```